.changes/unreleased/tinymce-TINY-11868-2025-05-12.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Cutting or deleting the whole element doesn't mutate other content.
```

You used present tense, fixes should be past tense. English can be hard 🙃 

And cut is just another form of delete so keep it simple 👍 
```suggestion
body: Deleting a whole element would sometimes modify nearby content.
```